src/dataset-builder/CohortEditor.ts

```diff
@@ -210,7 +208,7 @@ type AddCriteriaSelectorProps = {
   getNextCriteriaIndex: () => number;
   cohort: Cohort;
   domainOptions: DomainOption[];
-  programDataOptions: LoadedState<(ProgramDataListOption | ProgramDataRangeOption)[], unknown>;
+  programDataOptions: (ProgramDataListOption | ProgramDataRangeOption)[];
```

Comment:
Could you clarify the usage of `program` here? Just curious why it's important to include?

Reply:
naming convention wise .. what does it mean?